packages/cactus-plugin-ubiquity/src/main/typescript/prometheus-exporter/data-fetcher.ts

```diff
@@ -0,0 +1,11 @@
+import { Transactions } from "./response.type";
```

Comment:
The response.type.ts doesnt contain Transactions

Reply:
Fixed!